### PR TITLE
ENG-4259 recover enable save button code

### PR DESCRIPTION
**Pull request by ichalagashvili**

### CODE REVIEW

File: src/ui/widgets/config/WidgetConfigMicrofrontend.js [hunk 1]
```diff
@@ -25,6 +25,21 @@ const WidgetConfigMicrofrontend = ({
   const { assetLoading, mfe, hasError } = useMfe({ initialMfe: memoMfeObject });
   const customElement = get(mfe, 'customElement');
 
+  // eslint-disable-next-line no-unused-vars
+  const [enableSubmit, setEnableSubmit] = useState(true);
```
Comment (raxkaynan) [inline src/ui/widgets/config/WidgetConfigMicrofrontend.js:28]: `[, setEnableSubmit]` can be used here so the eslint comment can be removed.

Also, what's the purpose of using `setEnableSubmit()` if its corresponding state value isn't used?

**Review by raxkaynan — CHANGES_REQUESTED** [review]

File: src/ui/widgets/config/WidgetConfigMicrofrontend.js (hunk 1 above)
Comment (Dv92) [reply src/ui/widgets/config/WidgetConfigMicrofrontend.js:28]: @chalagashvili from what I see in the history, it's missing the `enableSubmit` binding to the save button. It's never been added 😅. So we need that property @raxkaynan but must be hook to the save button for disable when it is false.

**Review by Dv92 — CHANGES_REQUESTED** [review]
need the modification in the comment above

**Comment by Dv92** [comment]
moved to draft until 7.1.2 release

**Review by Dv92 — APPROVED** [review]

**Pull request merged**